wandb/wandb_torch.py

```diff
@@ -96,9 +92,6 @@ def add_log_hooks_to_pytorch_module(
         if name is not None:
             prefix = prefix + name
 
-        if jupyter_run:
```

Comment:
Im just curious. did you look into why this was here and what it was accomplishing before?

Reply:
I couldn't find what is was doing... I saw it is used to recover history... 
I guess I can keep it and just use it as a run...